.github/workflows/android.yml

```diff
@@ -0,0 +1,28 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: gradle
```

Comment:
Test whether the SDK version can help explain why the linter broke in Github Actions regarding the target version.

Reply:
We figured out what the problem was and discussed over zoom.  I have updated the lintOptions and either disabled the lint check or made it informational.

Reply:
https://github.com/simplymadeapps/PreferenceHelper/pull/12/commits/d280356cad852820edf535332f2393eeea8ec0f6

Reply:
https://github.com/stephenruda/PreferenceHelper/actions/runs/2266191556